utils/vara-ui/src/components/button/button.module.scss

```diff
@@ -1,46 +1,44 @@
-@keyframes blink {
-  0% {
-    background-position: -100% 0;
-  }
-
-  100% {
-    background-position: 100% 0;
-  }
-}
-
 .button {
-  border: none;
-  cursor: pointer;
-  padding: 0;
-  font-family: inherit;
-
   display: inline-flex;
   align-items: center;
   justify-content: center;
-  gap: 10px;
+  gap: 8px;
 
-  font-weight: 700;
-  line-height: 18px;
-  letter-spacing: 0.03em;
+  padding: 0;
 
-  border-radius: 2px;
-  transition: all 0.25s;
+  font-family: inherit;
+  font-weight: 600;
+  line-height: 24px;
+
+  border: none;
+  border-radius: 4px;
+  transition-property: color, background-color, box-shadow;
+  transition-duration: 250ms;
+
+  cursor: pointer;
+  outline: none;
+  outline-offset: 2px;
 
   &:disabled {
-    pointer-events: none;
+    cursor: not-allowed;
   }
 
-  /* standalone class cuz to omit isLoading */
-  &.disabled {
-    opacity: 0.3;
+  &:focus-visible {
+    outline: rgba(0, 255, 196, 1) 2px solid;
   }
 
-  &.loading {
-    animation: blink 1s linear infinite forwards;
-    background-size: 200%;
+  svg {
```

Comment:
I suggest to use class there and not general svg selector, to only target icons passed via `SVG` prop.

There can be cases when svg is passed via `children` prop, Identicon for example, for which size should not apply.

Also, I don't think that properties other than fixed width and height are useful there, for a size they are superior and should be enough. For color, we can't do much since `path` element is usually colored but it's not guaranteed, and also, some of the icons can have different color palette in terms of shades that we should't change.